code/modules/food_and_drinks/kitchen_machinery/smartfridge.dm

```diff
@@ -459,7 +459,9 @@
 	icon = 'icons/obj/kitchen.dmi'
 	icon_state = "foodcart"
 	anchored = FALSE
-	power_state = NO_POWER_USE
+	requires_power = FALSE
+	idle_power_consumption = 0
+	active_power_consumption = 0
```

Comment:
Don't think you need idle and active power set to 0, you can just use the NO_POWER_USE define with power state.

Reply:
`NO_POWER_USE` doesn't work, as mentioned. `requires_power = false` however does. Not sure why. But with `power_state = NO_POWER_USE` you're unable to access the smartfridge once power goes out if the cart is wrenched.

Though I believe you're right about idle and active not needing to be set to 0. It otherwise needs requires_power for it to work properly.

Reply:
> `NO_POWER_USE` doesn't work, as mentioned. `requires_power = false` however does. Not sure why. But with `power_state = NO_POWER_USE` you're unable to access the smartfridge once power goes out if the cart is wrenched.

power_state handles actually using power both on idle and active states, when set to NO_POWER_USE it will function like it has an idle and active power use of 0 without needed to fully go into procs. Requires power is what handles being able to use machines in areas with no power which was what was missing.

Reply:
Good to know. Removed the unneeded lines and re-added `power_state`.